src/views/common/TimeScrollView/TimeScrollViewInteractions/TimeScrollViewEventHandlers.ts

```diff
@@ -107,7 +107,11 @@ const useTimeScrollEventHandlers = (leftMargin: number, panelWidth: number, divR
     const handleMouseMove = useMouseMoveHandler(divRef, clickReader, startPan, setPanUpdate)
     const handleMouseLeave = useMouseLeaveHandler(divRef, clearPan)
 
-    return {handleMouseUp, handleMouseMove, handleMouseDown, handleMouseLeave}
+    const handlers = useMemo(() => {
```

Comment:
I don't know if this makes a difference, but it's nice to be explicit rather than return potentially referentially-unequal objects.